application/aam-backend-service/src/main/resources/application.yaml

```diff
@@ -67,7 +74,7 @@ sqs-client-configuration:
   basic-auth-password: docker
 
 database-change-detection:
-  enabled: true
+  enabled: false
```

Comment:
what changes are we not tracking here anymore? Schema changes? Doc changes triggering a report update?

Reply:
Just settings for the local development setup... Can be enabled by default again.